pootle/apps/pootle_store/forms.py

```diff
@@ -280,11 +284,16 @@ def clean_target_f(self):
 
             return value
 
-        def clean_state(self):
+        def _clean_state(self):
+            # Handle clean_state here because it depends on clean_suggestion.
             old_state = self.instance.state  # Integer
             is_fuzzy = self.cleaned_data['state']  # Boolean
             new_target = self.cleaned_data['target_f']
 
+            # If suggestion is provided set `old_state` should be `TRANSLATED`.
+            if self.cleaned_data['suggestion']:
+                old_state = TRANSLATED
+
             if (self.request is not None and
```

Comment:
for me putting `and` or other operators at the end of lines is not a matter of format but logic.

I know its not the pep8 spec, but these blocks of logic really do need to be understood when read, and i find the spec unreadable.

ignore if your really really want to 8/